i2c/README.md

```diff
@@ -0,0 +1,84 @@
+| `Class`          | X        | `i2c.relay.EightRelayHAT`       |                                                                                                                                                                                                                                                                               |
+| `Connections`    | X        | dictionary of connectors        | Defines where to subscribe for messages and where to publish the status for each connection. Look at connection readme's for 'Actuator / sensor relevant parameters' for details.                                                                                             |
+| `Stack`          |          | Stack Address 0..7              | Optional, the selected stack adress which can be set via jumpers on the header 0-1-2 (no jumper: Stack =  0, default = 0, see product description [here](https://sequentmicrosystems.com/collections/industrial-automation/products/8-relays-stackable-card-for-raspberry-pi))|
+| `Relay`          | X        | Relay No. 1..8                  | The relay to control                                                                                                                                                                                                                                                          |
+| `Level`          |          | DEBUG, INFO, WARNING, ERROR     | Override the global log level and use another one for this sensor.                                                                                                                                                                                                            |
+| `ToggleDebounce` |          | decimal number                  | The interval in seconds during which repeated toggle commands are ignored (default 0.15 seconds)                                                                                                                                                                              |
+| `InitialState`   |          | ON or OFF                       | Optional, when set to ON the pin's state is initialized to HIGH. Ignores InvertOut (default OFF)                                                                                                                                                                              |
+| `SimulateButton` |          | Boolean                         | When `True` simulates a button press by setting the pin to HIGH for half a second and then back to LOW. In case of `InitalState` ON it will toggle the other way around.                                                                                                      |
+
+
+
+
+### Outputs / Inputs
+
+The EightRelayHAT has only one output and input.
+The input expects ON, OFF, TOGGLE or a datetime string as command.
```

Comment:
I have no recollection why a date time string is treated as a toggle command. Does this still make sense?

I know you took that straight from the GPIO Actuator but I cannot for the life of me remember why it's there. It doesn't hurt to leave it and it's probably safest to do so, but I wonder if it supports a DT string, maybe that should be treated as the time to toggle the switch. Maybe a kind of way to schedule it for the future?

Regardless this isn't something to worry about for this PR. Just food for thought.

Reply:
The DT string is handled in this way to allow local connections with an `RpiGpioSensor`. When a `ShortButtonPress` or a `LongButtonPress` occurs, the `RpiGpioSensor` sends the current timestamp as DT string.  
This way it's very easy to debug the "toggle" events when using the other connectors on the server side (mqtt etc.), since a toggle event will changes the value. 
If the `RpiGpioSensor` would only sent `TOGGLE` the value would always stay the same and the user would have to figure out how to detect a value update.